src/lib/dev-environment/dev-environment-core.js

```diff
@@ -662,11 +669,10 @@ export async function getVersionList(): Promise<WordPressTag[]> {
 			await fs.promises.mkdir( cacheFilePath, { recursive: true } );
 		}
 
-		// If the cache doesn't exist, create it
-		// If the cache is expired, refresh it
-		if ( ! fs.existsSync( cacheFile ) || isVersionListExpired( cacheFile, DEV_ENVIRONMENT_WORDPRESS_VERSION_TTL ) ) {
+		// If the cache does not exist or has expired, refresh it
+		if ( await isVersionListExpired( cacheFile, DEV_ENVIRONMENT_WORDPRESS_VERSION_TTL ) ) {
```

Comment:
We do `stats()` in `isVersionListExpired()`, there's no sense to issue `existsSync()`, as it will be another call to `stats()`.